src/app/cli/src/coda.ml

```diff
@@ -148,6 +148,23 @@ let daemon logger =
            "true|false Limit the number of concurrent connections per IP \
             address (default:true)"
          (optional bool)
+     (*TODO: This is being added to log all the snark works received for the 
+     beta-testnet challenge. We might want to remove this later?*)
+     and log_received_snark_pool_diff =
+       flag "log-snark-work-gossip"
+         ~doc:
+           "true|false Log snark-pool diff received from peers (default:false)"
```

Comment:
all the other help strings have a space between "default:" and the value. this should match